src/bundler.js

```diff
@@ -259,6 +265,15 @@ function processJsBundle(options, jsBundle, bundleDir, jsFiles, bundleName, cb)
         };
 
         fs.writeFile(bundleName, allJs, afterBundle);
+
+        if (options.require) {

```

Comment:
We previously added the debug files one by one as we processed files in a given bundle. Since `require` bundles need to only have the combined file listed as their debug file, we don't want to do that anymore. We now add the debug files for a given bundle at the very end of processing the bundle, so we can easily do an `if`/`else` based on whether it's a `require` bundle.